CODEOWNERS

```diff
@@ -96,8 +96,8 @@
 /task/eaas-provision-space                                  @amisstea @avi-biton @hmariset @omeramsc @yftacherzog
 
 # renovate groupName=sealights
-/task/sealights-go         @ascerra @flacatus @psturc @dbaez
-/task/sealights-python     @ascerra @flacatus @psturc @dbaez
+/task/sealights-go         @ascerra @flacatus @psturc @Dannyb48
+/task/sealights-python     @ascerra @flacatus @psturc @Dannyb48
```

Comment:
This needs a corresponding update, otherwise LGTM

Reply:
@chmeliik done!